source/systemUtils.py

```diff
@@ -60,6 +60,34 @@ def hasUiAccess():
 		ctypes.windll.kernel32.CloseHandle(token)
 
 
+TokenOrigin = 17
+
+
+class TOKEN_ORIGIN(ctypes.Structure):
+	_fields_ = [("OriginatingLogonSession", ctypes.c_ulonglong)]
+
+
+def getProcessTokenOrigin(processHandle):
+	token = ctypes.wintypes.HANDLE()
+	ctypes.windll.advapi32.OpenProcessToken(
```

Comment:
This call could fail due to not enough permissions on the process handle. Perhaps return None or raise an exception if it returns false